buildpacks/python/tests/detect_test.go

```diff
@@ -4,81 +4,191 @@
 package tests
 
 import (
-	"bytes"
-	"kn-fn/python-function-buildpack/python"
-	"reflect"
+	"os"
 	"testing"
 
 	"github.com/buildpacks/libcnb"
-	"github.com/paketo-buildpacks/libpak/bard"
+	. "github.com/onsi/gomega"
```

Comment:
Unless this is avoiding a circular dependency, let's avoid `.` importing in Go. I think Gomega docs suggest using `gomega.Register...` for example :D

Reply:
Sounds good. This is the only pattern I've used for Gomega, so I'm super interested to check out `Register` 👍

Reply:
Hmm... I'm not seeing any `Register` function in the gomega docs. Could you link to what you were referring to?

Without the `.` import, assertions become a bit less readable:

```go
gomega.Expect(ACTUAL).To(gomega.Equal(EXPECTED))
```

It could be mitigated with a short alias like `g`:

```go
g.Expect(ACTUAL).To(g.Equal(EXPECTED))
```

I tend to avoid `.` imports too, though I might argue that we make an exception in this case and allow it because:

- It's test code vs prod code
- Chances of collision with the gomega API in test code is very small (zero if we aren't doing any other `.` imports)
- Readability for tests is especially important

Thoughts?

Reply:
I'm personally ok with `g.Expect(ACTUAL).To(g.Equal(EXPECTED))` because it follows Go's recommendations better, but I take it many won't be. Looking at it, as long as our files are postfixed with `_test.go`, which all are except `containers.go`, I have no qualm with using `.` notation for the Gomega import now.

Reply:
So looks like we're mostly in good form anyway to use the `.` notation -- since you'rew right -- this is only in `/tests`, it's not a package/lib used by others, and human readability is important. :D